Gemfile

```diff
@@ -70,7 +70,7 @@ group :test do
   gem 'rack-test', require: 'rack/test'
   gem 'mocha', require: false
   gem 'bourne', require: false
-  gem 'shoulda', require: false
+  gem 'shoulda'
```

Comment:
Why are we removing `require: false`?

Reply:
Updated commit msg to expand on version bump